qutebrowser/browser/commands.py

```diff
@@ -620,30 +620,35 @@ def scroll_page(self, x: float, y: float, *,
                 "representation.")
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
-    def yank(self, title=False, sel=False, domain=False, pretty=False):
-        """Yank the current URL/title to the clipboard or primary selection.
+    @cmdutils.argument('what', choices=['selection', 'url', 'pretty-url',
+                                        'title', 'domain'])
+    def yank(self, what='url', sel=False, keep=False):
+        """Yank something to the clipboard or primary selection.
 
         Args:
+            what: What to yank.
```

Comment:
Maybe the choices should be described here, similar to `start()` in `browser/hints.py`? I wondered what `selection` was before looking at the code.